atlp-rwanda/c4-barefoot-frontend
# Feature: application usage report feature

# Description

Dear colleagues,
this PR is for adding the feature of `application usage report / statistics` just as on the [mockup](https://www.figma.com/file/hM1DgjHISuEDVenacmEyo7/Barefoot-Nomad?node-id=125%3A61).

## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- Clone this repo
- Run the command `npm i`
- Run the command `npm run dev`
- Once the application launches in the browser, open the route `/admin` to view the [`application usage feature`](https://trello.com/c/lW99gnWb/51-application-usage-report), then you should be able to see all the mentioned tasks implemented.
# Checklist:


# Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/54407341/113284231-2f3dcf00-929e-11eb-82b2-80f080e6a5e1.png)
